src/library/dlgcoverartfullsize.cpp

```diff
@@ -46,6 +46,12 @@ DlgCoverArtFullSize::DlgCoverArtFullSize(QWidget* parent, BaseTrackPlayer* pPlay
     setupUi(this);
 }
 
+void DlgCoverArtFullSize::closeEvent(QCloseEvent* event) {
+    // prevent the window from being destroyed
+    hide();
```

Comment:
`slotLoadTrack(nullptr);`

Otherwise a reference to the current `Track` object is kept and connections remain active.

Resetting the track pointer currently does not reset the pixmap. Not sure how to handle this.